gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
te pliki wygenerowały się z wrapperem. Możesz je usunąć. Jak nie używasz gralde wrappera to one nie będą się generować